Source/MainWindow.cpp

```diff
@@ -278,7 +279,7 @@ MainWindow::AboutRequested()
 		NULL
 	};
 	
-	about->AddDescription(B_TRANSLATE("Open source weather app."));
+	about->AddDescription(B_TRANSLATE("Open source weather app for Haiku."));

```

Comment:
I'd remove the "for Haiku". If you see the about box you have to be running Haiku, which implies that Weather does actually run on Haiku, of course...

How about being more descriptive:
"A weather app showing a forecast for the next 5 days."